service/collector/helper.go

```diff
@@ -46,11 +55,36 @@ func newHelper(config helperConfig) (*helper, error) {
 		return nil, microerror.Maskf(invalidConfigError, "%T.AWSConfig must not be empty", config)
 	}
 
+	var clusterStore = cache.NewStore(cache.DeletionHandlingMetaNamespaceKeyFunc)
+	var configStore = cache.NewStore(cache.DeletionHandlingMetaNamespaceKeyFunc)
+	{
+		listerWatcher := &ClusterListerWatcher{
+			clusters: config.G8sClient.InfrastructureV1alpha2().AWSClusters(metav1.NamespaceAll),
+		}
+		reflector := cache.NewReflector(listerWatcher, &v1alpha2.AWSCluster{}, clusterStore, 2*time.Minute)
+		go reflector.Run(make(<-chan struct{}))
+		// force 1st reflector sync
+		listerWatcher.List(metav1.ListOptions{})
+	}
+
```

Comment:
Nit: It's easier to read _to me_ that way.

```suggestion

	var configStore = cache.NewStore(cache.DeletionHandlingMetaNamespaceKeyFunc)
```